packages/core/src/sass/app/_common/layout/header.scss

```diff
@@ -382,3 +382,43 @@
         }
     }
 }
+
+.header-v2 {
+    display: flex;
```

Comment:
Not sure, but maybe new header should be 48px?
![Screenshot 2024-05-10 at 1 32 55 PM](https://github.com/binary-com/deriv-app/assets/121025168/0c59e6bc-3674-47f8-b86c-7468dedc4a19)

Reply:
Cause currently is's 40

Reply:
Please, also check footer nav height

Reply:
Sure. i did not put much time/thought into header since it'll more than likely change in design.